packages/components/src/alert/Alert.spec.js

```diff
@@ -1,144 +1,250 @@
+      ({ container, getByText } = render(<Alert action={action} message={message} />));
+      component = getByRole(container, 'alert');
 
-      expect(component.hasClass('arrow')).toBe(true);
-      expect(component.hasClass('arrow-left')).toBe(false);
+      const el = getByText(action.text);
+      expect(el.attributes.getNamedItem('aria-label').value).toBe(action['aria-label']);
+      expect(el.attributes.getNamedItem('href').value).toBe(action.href);
     });
+  });
+
+  describe('markdown support', () => {
+    const input = 'That is one **bold cat**';
+    const output = 'That is one <strong>bold cat</strong>';
 
-    it('checks center top arrow', () => {
-      component.setProps({ arrow: Alert.ArrowPosition.TOP });
+    it('converts message to markdown', () => {
+      ({ container } = render(<Alert message={input} />));
+      component = getByRole(container, 'alert');
 
-      expect(component.hasClass('arrow')).toBe(true);
-      expect(component.hasClass('arrow-center')).toBe(true);
+      expect(component).toContainHTML(output);
     });
 
-    it('checks right top arrow', () => {
-      component.setProps({ arrow: Alert.ArrowPosition.TOP_RIGHT });
+    it('does not convert children to markdown', () => {
+      ({ container, getByText } = render(<Alert>{input}</Alert>));
+      component = getByRole(container, 'alert');
 
-      expect(component.hasClass('arrow')).toBe(true);
-      expect(component.hasClass('arrow-right')).toBe(true);
+      getByText(input);
+      expect(component).not.toContainHTML(output);
     });
+  });
+
+  describe('className', () => {
+    it('applies provided classes', () => {
+      ({ container } = render(<Alert className="cats" message={message} />));
+      component = getByRole(container, 'alert');
+      expect(component.className).toContain('cats');
+    });
+  });
+
+  describe('custom icon', () => {
+    it('uses any provided icon in preference to the default', () => {
+      const icon = <Emoji size={24} />;
+      const iconHTML = renderIcon(Emoji);
 
-    it('checks left bottom arrow', () => {
-      component.setProps({ arrow: Alert.ArrowPosition.BOTTOM_LEFT });
+      ({ container } = render(<Alert icon={icon} message={message} />));
+      component = getByRole(container, 'alert');
 
-      expect(component.hasClass('arrow')).toBe(true);
-      expect(component.hasClass('arrow-left')).toBe(true);
-      expect(component.hasClass('arrow-bottom')).toBe(true);
+      expect(component).toContainHTML(iconHTML);
+      expect(component).not.toContainHTML(iconTypeMap[Alert.Type.NEGATIVE]);
     });
+  });
+
+  describe('onDismiss', () => {
+    it('renders the close button if onDismiss is provided', () => {
+      const onDismiss = jest.fn();
 
-    it('checks center bottom arrow', () => {
-      component.setProps({ arrow: Alert.ArrowPosition.BOTTOM });
+      ({ container } = render(<Alert onDismiss={onDismiss} message={message} />));
```

Comment:
import { screen } from '@testing-library/dom';

```suggestion
      render(<Alert onDismiss={onDismiss} message={message} />);

      expect(screen.getByRole('button')).toBeTruthy();
```